### ISSUE TITLE
The "Download CR3" button does not populate a .PDF when you click it. 

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#11280]
> What application are you using?

Vision Zero (Editor)

> Describe the problem.

The "Download CR3" button does not populate a .PDF when you click it. 

> Is there anything else we should know?

This one is also having the same issue: 
https://visionzero.austin.gov/editor/#/crashes/19330991

> Website Address

https://visionzero.austin.gov/editor/#/crashes/19151437

Internet Browser: Edge

> Requested By
Xavier A.

Request ID: DTS23-106255

**Comment by patrickm02L** [comment]
Hoping that this is an HTML login page issue like in #10999
And not

**Comment by mddilley** [comment]
@patrickm02L Thanks for the resources in this issue. The CR3s are now available for these two records in VZE. Steps that I completed to fix this:

1. Deleted files for `19151437` and `19330991` from the `atd-vision-zero-editor/production/cris-cr3-files/` S3 bucket
2. Updated records with `crash_id` of `19151437` and `19330991` in the production VZD using the statement below. This reset the data that the CR3 script looks for when it decides whether to download the CR3 and then upload it to S3.
```sql
UPDATE atd_txdot_crashes 
SET cr3_stored_flag = 'N', cr3_file_metadata = null, cr3_ocr_extraction_date = null
WHERE crash_id = <the crash ID>;
```
3. Ran the CR3 download script to fill in these two records and upload their CR3 pdfs.
4. Tested in VZE
